airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/exceptions.py

```diff
@@ -49,3 +49,5 @@ class BulkJobAccessDenied(BaseBulkException):
 
     class BulkJobConcurrentError(BaseBulkException):
         """Raised when BULK Job could not be created, since the 1 Bulk job / shop quota is exceeded."""
+
+        failure_type: FailureType = FailureType.transient_error
```

Comment:
This error is currently flagged as config error but the user can't do anything about this. This is actually a rate limiting issue and as such, I wanted to update the failure_type

Reply:
Thank you for this change.